lib/train/extras/os_common.rb

```diff
@@ -96,6 +98,8 @@ def detect_family
       # TODO: extend base implementation for detecting the family type
       # to Windows and others
       case uname_s
+      when /unrecognized command verb/
```

Comment:
Okay, lets use this for now until we find any other `non-uname` operating system